src/skin/popup.html

```diff
@@ -128,11 +128,24 @@ <h2 id="title-name"><span class="i18n_name"></span></h2>
       <span class="i18n_popup_instructions_no_trackers" data-i18n_contents_placeholders="<a target='_blank' title='i18n_what_is_a_tracker' class='tooltip' href='https://privacybadger.org/#What-is-a-third-party-tracker'>"></span>
       <span id="no-third-parties" class="i18n_popup_blocked" style="display:none"></span>
     </span>
+    <div id="toggle-blocked-resources-container">
+      <span id="expand-blocked-resources" class="ui-icon ui-icon-caret-d"></span>
```

Comment:
This caret is visible on pages with no third parties (`https://example.com`).

Reply:
All carets should get the hand cursor like other clickable elements.

Reply:
Should carets get a tooltip? "Click to show more information"?

Reply:
Whatever ends up being clickable to expand/collapse sections should be keyboard-accessible. Users should be able to <kbd>Tab</kbd> to visibly select each toggle element and <kbd>Space</kbd> (or whatever the standard key is) to activate.

Reply:
Resolved all these except the accessible activation via the keyboard bit. At its current working state, one can focus on the element from the keyboard, but I'm unable to get the activation bit working. I tried attaching on `keydown` listeners and more tradition `onClick`s to it, no luck. On the other hand, while testing this out I realized there are a few places that aren't accessible or end up "trapping" the user into some modals that they can't easily exit out of without a mouse action. Perhaps that's worth addressing in a follow up PR?

Reply:
`<div>` or `<span>` elements don't have built-in keyboard accessibility; '<a>' and `<button>` elements do.

To make a `<span>` focusable and so able to receive keyboard events, you have to add a `tabindex=0`. You then have to add a `keydown` listener and filter for supported keys.

Make sure to add the `tabindex` property to the same element that your `keydown` listener is on.

Reply:
You'll also want to add proper ARIA attributes (such as `role=button` perhaps) to your elements.

Reply:
You could instead use a `<button>` or an `<a href="" role="button">` element, which automatically support keyboard navigation and also trigger your click handler on <kbd>Space</kbd>/<kbd>Enter</kbd> presses. You will probably want to `e.preventDefault()` inside the click handler (to avoid performing the unwanted default action of `<a>`).

Reply:
Yes to improving keyboard navigation in a followup PR: https://github.com/EFForg/privacybadger/pull/2629#issuecomment-646101693